docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -6,44 +6,48 @@
 
-### Quick Start
+!!! Tip "Quick Start"
+    Spinning up EdgeX with snaps is extremely easy.
+    For demonstration purposes, let's install the platform, along with the virtual device service and EdgeX UI.
 
-Spinning up EdgeX with snaps is extremely easy.
-For demonstration purposes, let's install the platform, along with the virtual device service and EdgeX UI.
+    1) Install the [platform snap], [Device Virtual](#device-virtual) and [EdgeX UI](#edgex-ui):
+    ```bash
+    snap install edgexfoundry edgex-device-virtual edgex-ui
+    ```
+    This installs the latest stable version of the snaps. The [installation](#installation) section provides more explanations.
 
-1) Install the [platform snap], [Device Virtual](#device-virtual) and [EdgeX UI](#edgex-ui):
-```bash
-snap install edgexfoundry --edge
-snap install edgex-device-virtual --edge
-snap install edgex-ui --edge
-```
-!!! danger
-    The above examples use the latest development versions of EdgeX snaps.
-    This should be replaced by the latest stable v3, once available.
 
-EdgeX has security enabled by default. But for simplicity, we'll disable it here.
-If not disabled, you could skip to step 3 and additionally add a user and authenticate as described [here](#adding-api-gateway-users). To access the UI, you need to generate a JWT.
+    2) Disable security in each of the installed snaps:
+    ```bash
+    snap set edgexfoundry security=false
+    snap set edgex-device-virtual config.edgex-security-secret-store=false
+    snap set edgex-ui config.edgex-security-secret-store=false
+    ```
 
-2) Disable security in each of the installed snaps:
-```bash
-snap set edgexfoundry security=false
-snap set edgex-device-virtual config.edgex-security-secret-store=false
-snap set edgex-ui config.edgex-security-secret-store=false
-```
-Refer to [disabling security] for details.
+    Beware that this leaves the services at risk! We do it here only to simplify the quick start.
+    Refer to [disabling security] for details.
 
-3) Start the device and UI services:
-```bash
-snap start edgex-device-virtual
-snap start edgex-ui
-```
-This is because those services disabled and not started by default.
+    3) Start the services:
+    ```bash
+    # start Core and Support services in the platform snap
+    sudo snap start edgexfoundry.consul edgexfoundry.redis \
+        edgexfoundry.core-common-config-bootstrapper \
+        edgexfoundry.core-data edgexfoundry.core-metadata edgexfoundry.core-command \
+        edgexfoundry.support-scheduler edgexfoundry.support-notifications
 
-You should now be able to access the UI using a browser at [http://localhost:4000](http://localhost:4000)
+    # start Device Virtual
+    snap start edgex-device-virtual
+
+    # start EdgeX UI
+    snap start edgex-ui
+    ```
 
-![EdgeX UI](EdgeX-GettingStartedSnapUsersUI.png)
+    You should now be able to access the UI using a browser at [http://localhost:4000](http://localhost:4000)
```

Comment:
After turning security off, the UI is still requesting a token for login. Additionally, an error in the log:
```
2023-05-17T17:38:11+02:00 edgex-ui.edgex-ui[316315]: level=INFO ts=2023-05-17T15:38:11.923881812Z app=edgex-ui-service source=httpserver.go:133 msg="Web server starting (localhost:4000)"
2023-05-17T17:38:11+02:00 edgex-ui.edgex-ui[316315]: level=INFO ts=2023-05-17T15:38:11.923892801Z app=edgex-ui-service source=message.go:50 msg="Service dependencies resolved..."
2023-05-17T17:38:11+02:00 edgex-ui.edgex-ui[316315]: level=INFO ts=2023-05-17T15:38:11.923898743Z app=edgex-ui-service source=message.go:51 msg="Starting edgex-ui-service 3.0.0-dev.10 "
2023-05-17T17:38:11+02:00 edgex-ui.edgex-ui[316315]: level=INFO ts=2023-05-17T15:38:11.923903258Z app=edgex-ui-service source=message.go:55 msg="edgex-ui-go service started"
2023-05-17T17:38:11+02:00 edgex-ui.edgex-ui[316315]: level=INFO ts=2023-05-17T15:38:11.923909375Z app=edgex-ui-service source=message.go:58 msg="Service started in: 1.198248ms"
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/runtime.a3ca446c1cda35cc71f0.js
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/polyfills.ee9e4a31e0a58f0dd9bc.js
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/scripts.c7db3b1843cdd546a4ba.js
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/main.1ee8a1b412e3fea92fe7.js
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/styles.2a143380ce0f2a41afa9.css
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/styles.2a143380ce0f2a41afa9.css
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/assets/img/bg_x.png
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /en-US/fontawesome-webfont.20fd1704ea223900efa9.woff2
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: /core-metadata/api/v3/ping
2023-05-17T17:38:19+02:00 edgex-ui.edgex-ui[316315]: 2023/05/17 17:38:19 http: proxy error: dial tcp 127.0.0.1:8000: connect: connection refused
```

Reply:
Fix for the security (and other env overrides): https://github.com/edgexfoundry/edgex-ui-go/pull/613

Reply:
I have tested the snap built from the above PR and now the UI can be accessed in security off mode.